tool.drsyscall-record-trimmer-test fails with -1 exit code on newer distro

On our internal Linux distro, which has more recent kernel+glibc+etc. compared to the GA Vms, the trimmer test fails:
```
$ ctest -V -R trimmer-test
UpdateCTestConfiguration  from :$HOME/dr/git/build_x64_dbg_tests/DartConfiguration.tcl
Parse Config file:$HOME/dr/git/build_x64_dbg_tests/DartConfiguration.tcl
Test project $HOME/dr/git/build_x64_dbg_tests
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 151
    Start 151: code_api|tool.drsyscall-record-trimmer-test

151: Test command: /usr/bin/cmake "-D" "precmd=" "-D" "cmd=$HOME/dr/git/build_x64_dbg_tests/bin64/drrun@-s@90@-quiet@-debug@-killpg@-stderr_mask@0xC@-dumpcore_mask@0@-code_api@--@$HOME/dr/git/build_x64_dbg_tests/suite/tests/bin/simple_app" "-D" "postcmd=$HOME/dr/git/build_x64_dbg_tests/bin64/drsyscall_record_viewer@$HOME/dr/git/build_x64_dbg_tests/suite/tests/syscall_record_file.test" "-D" "postcmd2=$HOME/dr/git/build_x64_dbg_tests/bin64/drsyscall_record_trimmer@-input_file@$HOME/dr/git/build_x64_dbg_tests/suite/tests/syscall_record_file.test@-output_file@$HOME/dr/git/build_x64_dbg_tests/suite/tests/syscall_record_file.test.trimmed@-trim_before_timestamp@10012@-trim_after_timestamp@10015" "-D" "postcmd3=$HOME/dr/git/build_x64_dbg_tests/bin64/drsyscall_record_viewer@$HOME/dr/git/build_x64_dbg_tests/suite/tests/syscall_record_file.test.trimmed" "-D" "postcmd4=" "-D" "failok=" "-D" "cmp=$HOME/dr/git/build_x64_dbg_tests/suite/tests/drsyscall-record-trimmer-test.expect" "-D" "code=" "-D" "capture=" "-D" "ignore_matching_lines=" "-P" "$HOME/dr/git/src/suite/tests/runmulti.cmake"
151: Working Directory: $HOME/dr/git/build_x64_dbg_tests/suite/tests
151: Test timeout computed to be: 90
151: Running cmd |$HOME/dr/git/build_x64_dbg_tests/bin64/drrun;-s;90;-quiet;-debug;-killpg;-stderr_mask;0xC;-dumpcore_mask;0;-code_api;--;$HOME/dr/git/build_x64_dbg_tests/suite/tests/bin/simple_app|
151: Running postcmd |$HOME/dr/git/build_x64_dbg_tests/bin64/drsyscall_record_viewer;$HOME/dr/git/build_x64_dbg_tests/suite/tests/syscall_record_file.test|
151: CMake Error at $HOME/dr/git/src/suite/tests/process_cmdline.cmake:109 (message):
151:   *** postcmd failed (255): ***
151:
151: Call Stack (most recent call first):
151:   $HOME/dr/git/src/suite/tests/runmulti.cmake:60 (process_cmdline)
151:
151:
1/1 Test #151: code_api|tool.drsyscall-record-trimmer-test ...***Failed    0.50 sec
```

--- a/suite/tests/CMakeLists.txt
+++ b/suite/tests/CMakeLists.txt
@@ -6793,9 +6793,10 @@ IF (NOT ANDROID AND NOT APPLE AND NOT MUSL AND NOT RISCV64 AND NOT WINDOWS)
     drsyscall_record_lib)
 
   if (X64)
-    file(COPY syscall_record_file.test DESTINATION "${PROJECT_BINARY_DIR}/suite/tests")
     set(syscall_record_test_file
-      "${PROJECT_BINARY_DIR}/suite/tests/syscall_record_file.test")
+      # Add a "trim_" prefix to avoid deletion by client.syscall-records-test.
+      "${PROJECT_BINARY_DIR}/suite/tests/trim_syscall_record_file.test")
+    file(COPY_FILE syscall_record_file.test "${syscall_record_test_file}")
     set(tool.drsyscall-record-trimmer-test_runcmp
       "${CMAKE_CURRENT_SOURCE_DIR}/runmulti.cmake")
     set(tool.drsyscall-record-trimmer-test_expectbase "drsyscall-record-trimmer-test")

--- a/suite/tests/CMakeLists.txt
+++ b/suite/tests/CMakeLists.txt
@@ -6796,7 +6796,7 @@ IF (NOT ANDROID AND NOT APPLE AND NOT MUSL AND NOT RISCV64 AND NOT WINDOWS)
     set(syscall_record_test_file
       # Add a "trim_" prefix to avoid deletion by client.syscall-records-test.
       "${PROJECT_BINARY_DIR}/suite/tests/trim_syscall_record_file.test")
-    file(COPY_FILE syscall_record_file.test "${syscall_record_test_file}")
+    configure_file(syscall_record_file.test "${syscall_record_test_file}" COPYONLY)
     set(tool.drsyscall-record-trimmer-test_runcmp
       "${CMAKE_CURRENT_SOURCE_DIR}/runmulti.cmake")
     set(tool.drsyscall-record-trimmer-test_expectbase "drsyscall-record-trimmer-test")

